server/api/hot_status.go

```diff
@@ -131,13 +132,23 @@ func (h *hotStatusHandler) GetHotStores(w http.ResponseWriter, r *http.Request)
 		QueryWriteStats: make(map[uint64]float64),
 		QueryReadStats:  make(map[uint64]float64),
 	}
-	for id, loads := range h.GetStoresLoads() {
-		stats.BytesWriteStats[id] = loads[statistics.StoreWriteBytes]
-		stats.BytesReadStats[id] = loads[statistics.StoreReadBytes]
-		stats.KeysWriteStats[id] = loads[statistics.StoreWriteKeys]
-		stats.KeysReadStats[id] = loads[statistics.StoreReadKeys]
-		stats.QueryWriteStats[id] = loads[statistics.StoreWriteQuery]
-		stats.QueryReadStats[id] = loads[statistics.StoreReadQuery]
+	stores, _ := h.GetStores()
+	storesLoads := h.GetStoresLoads()
+	for _, store := range stores {
+		id := store.GetID()
```

Comment:
it will be better if use stores map<storeID,loads> returns

Reply:
The format of the api return result cannot be changed.